build.sbt

```diff
@@ -29,15 +29,24 @@ val commonSettings: Seq[SettingsDefinition] = Seq(
 resolvers += Resolver.sonatypeRepo("releases")
 
 
-// This project is compiled against multiple versions of `play-json` to aid compatibility.
-// The root build is published as `acquisition-event-producer` and uses play-json 2.4;
-// we also publish builds for play-json 2.5 and 2.6. The directories for these builds
-// (`play25`, `play25` etc) aren't real files but need to be specified in this way to
-// emulate separate projects.
 lazy val root = (project in file("."))
+  .aggregate(eventProducerPlayJson27, eventProducerPlayJson28)
+  .settings(
+    sourceDirectory := baseDirectory.value / "dummy",
```

Comment:
why do we need this?

Reply:
I think (from memory) if not it was compiling the source code under `root` project as well, because the source code is actually in the root.
I based it partly on a previous version that handled play 2.5 and 2.6, but that worked in a slightly different way.